### ISSUE TITLE
Failing tests after upgrading UI dependencies

### ISSUE THREAD

**Issue by antgamdia** [kubeapps/kubeapps#3162]
TBD

### Description:

Last time we tried it, we got an error, that, presumably, has something to do with a transitive dependency.

### Steps to reproduce the issue:

1. `yarn test` -> ok
2. `yarn upgrade`
3. `yarn test` -> nok

### Describe the results you received:

Tests failing in `AppView` related views when detecting the cluster being passed
```
    @@ -1,9 +1,9 @@
      Array [
        ResourceRef {
          "apiVersion": "apps/v1beta1",
    -     "cluster": "cluster-1",
    +     "cluster": undefined,
    > 403 |     expect(applicationStatus.prop("statefulsetRefs")).toEqual([
 ```

### Describe the results you expected:

Tests passing OK

### Additional information you deem important (e.g. issue happens only occasionally):

N/A

### Version of Helm, Kubeapps and Kubernetes:

N/A (dev)

**Comment by antgamdia** [comment]
The PR https://github.com/kubeapps/kubeapps/pull/3163 is performing this upgrade so that you can see the error in the CI logs. 
I'll eventually return to this issue to see what's happening.

**Comment by dlaloue-vmware** [comment]
i tried this on my machine.
I did a yarn test (successful) then a yarn upgrade and re-ran yarn test (failed).
I had 2 failures with the following message: Cannot find module 'isomorphic-form-data' from 'node_modules/swagger-ui-react/swagger-ui.js'

after reverting my changes, I added the following line to package.json: "isomorphic-form-data": "^2.0.0".
I ran yarn install, ran yarn test successfully, then ran yarn upgrade again and the following yarn test was successful.

i did notice that the module "isomorphic-form-data" was not downloaded in the nodes_modules directory until i make the changes to package.json.

Antonio, does this seem to make sense to you?
